DelvEdit/src/com/interrupt/dungeoneer/editor/file/EditorFile.java

```diff
@@ -349,21 +349,7 @@ public void onDontSave() {
     }
 
     private void createInternal(int width, int height) {
```

Comment:
I didn't want to destroy the API here. But I think it is nowhere else used?!

Reply:
It is only used by `EditorFile.create()`. I think this change is fine. :)